packages/tocco-ui/src/EditableValue/typeEditorFactory.js

```diff
@@ -86,12 +87,12 @@ export default (type, value, onChange, options, id, events, readOnly = false) =>
     }
 
     return (
-      <div {...events}>
+      <div {..._omit(events, 'onChange')}>
         <Component
           value={value}
           onChange={v => {
             blurValue = v
-            onChange(v)
+            events.onChange(v)
           }}
           {...(_isEmpty(options) ? {} : {options})}
```

Comment:
why not `{...(_isEmpty(options) ? options : {options})}` ?

Reply:
not easier to read IMO